File: easy/java/src/StringTask.java
//https://codeforces.com/problemset/problem/118/A

import java.util.Scanner;

public class StringTask {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String givenString = scanner.nextLine().toLowerCase();
        char[] vowels = {'a', 'o', 'y', 'e', 'u', 'i'};
        
        StringBuilder result = new StringBuilder();
        for (char c : givenString.toCharArray()) {
            boolean isVowel = false;
            for (char vowel : vowels) {
                if (c == vowel) {
                    isVowel = true;
                    break;
                }
            }
            if (!isVowel) {
                result.append(".").append(c);
            }
        }
        
        System.out.println(result.toString());
        scanner.close();
    }
}

File: codeforces/python/string_task.py
#https://codeforces.com/problemset/problem/118/A

given_string = input().lower()
vowels = ["a", "o", "y", "e", "u", "i"]

result = ""
for i in given_string:
    if i not in vowels:
        result += "." + i
        
print(result)
